--- eyeD3-0.8/src/eyed3/plugins/__init__.py.orig	2017-03-12 23:18:52.000000000 +0100
+++ eyeD3-0.8/src/eyed3/plugins/__init__.py	2019-02-24 18:04:20.190456261 +0100
@@ -50,11 +50,10 @@
 
     def _isValidModule(f, d):
         '''Determine if file ``f`` is a valid module file name.'''
-        # 1) tis a file
+        # 1) is a file
         # 2) does not start with '_', or '.'
-        # 3) avoid the .pyc dup
         return bool(os.path.isfile(os.path.join(d, f)) and
-                    f[0] not in ('_', '.') and f.endswith(".py"))
+                    f[0] not in ('_', '.') and (f.endswith(".py") or f.endswith(".pyc")))
 
     log.debug("Extra plugin paths: %s" % paths)
     for d in [os.path.dirname(__file__)] + (paths if paths else []):
@@ -65,11 +64,16 @@
         if d not in sys.path:
             sys.path.append(d)
         try:
+            loaded = set()
             for f in os.listdir(d):
                 if not _isValidModule(f, d):
                     continue
 
                 mod_name = os.path.splitext(f)[0]
+                if mod_name in loaded:
+                    continue
+                loaded.add(mod_name)
+
                 try:
                     mod = __import__(mod_name, globals=globals(),
                                      locals=locals())
